src/components/text-input/index.md.njk

```diff
@@ -31,6 +31,14 @@ All text inputs must have visible labels; placeholder text is not an acceptable
 
 Labels should be aligned above the text input they refer to. They should be short, direct and written in sentence case. Don’t use colons at the end of labels.
 
+### Label as a page heading
+
+When asking one thing per page you may want your `<label>` to be the page  heading. The Nunjucks macro makes this easy by adding the `<h1>` for you when you set `isPageHeading: true,`.
```

Comment:
This talks about the 'one thing per page' principle but the labels example doesn't – is that intentional?